docs/resources/samples/getting-started.md

```diff
@@ -118,7 +118,7 @@ You can use functions to organize and reuse code within your script.
 
 ### Objects, hierarchy, methods, properties, collections
 
-All of the Excel's object model is defined in a hierarchical structure of objects, beginning with the workbook object of type `ExcelScript.Workbook`. An object can contain methods, properties, and other objects within it. Objects are linked to each other using the methods. An object's method can return another object or collection of objects. Using the code editor's IntelliSense (code completion) feature is a great way to explore the object hierarchy. You can also use the [official reference documentation site](/javascript/api/office-scripts/overview?view=office-scripts) to follow along with the relationships among objects.
+All of the Excel's object model is defined in a hierarchical structure of objects, beginning with the workbook object of type `ExcelScript.Workbook`. An object can contain methods, properties, and other objects within it. Objects are linked to each other using the methods. An object's method can return another object or collection of objects. Using the code editor's IntelliSense (code completion) feature is a great way to explore the object hierarchy. You can also use the [official reference documentation site](/javascript/api/office-scripts/overview?view=office-scripts&preserve-view=true) to follow along with the relationships among objects.
```

Comment:
I don't think we need the `preserve-view` param any more, not that the other moniker is deprecated.

Reply:
so can I just remove the ?view param altogether? As it stands, I can't just leave ?view-office-scripts because we get a build warning

Reply:
Yes, we can remove that all together. Thanks!